src/components/MultipleLeftRightSelector/ValueList/ValueList.tsx

```diff
@@ -0,0 +1,93 @@
+import React, {useState} from 'react';
+import {ListItem, Typography, SearchInput, FAKE_VALUE} from '~/components';
+import {ChevronRight, Close, HandleDrag} from '~/icons';
+import cslx from 'clsx';
+import {ValueListProps, Value} from './ValueList.types';
+import './ValueList.scss';
+import clsx from 'clsx';
+
+export const ValueList: React.FC<ValueListProps> = ({
+    values,
+    onMove,
+    orientation,
+    draggedId,
+    readOnly,
+    iconStartProps = () => ({}),
+    listItemProps = () => ({})
+}) => {
+    const [filter, setFilter] = useState(null);
+
+    const iconProp = (v: Value) => {
+        const filterProp = filter ? {} : iconStartProps(v);
+
+        if (orientation === 'left') {
+            return {
+                iconEnd: readOnly ? null : (
+                    <div className="moonstone-iconContainer">
+                        <ChevronRight className="moonstone-displayNone"/>
+                    </div>
+                ),
+                iconStart: readOnly ? null : (
+                    <div className="moonstone-iconContainer" {...filterProp}>
+                        <HandleDrag/>
+                    </div>
```

Comment:
As we don't allow drag in the left list, we should not display the `HandleDrag` icon.